mutatex/utils.py

```diff
@@ -208,6 +212,10 @@ def parse_poslist_file(fname):
             log.error("the position list file is not in the right format")
             log.error("format error at %s" % line.strip())
             raise TypeError
+        if residue not in unique_residues:
+            if line.strip("\n") not in str(unique_residues):
```

Comment:
I'm not sure what's the purpose of this double if. Especially, the second part makes the check fragile (i.e. what if I have  `IA15` in the poslist and my PDB also contains residue `IA150`?)